format/FormatNumber.ts

```diff
@@ -53,6 +53,9 @@ export interface NumberFormatOptions extends Omit<FormatOptions<number>, 'toolti
     /** True to prepend positive numbers with a '+'. */
     withPlusSign?: boolean;
 
+    /** True to not display a signed zero -0.0 (and +0.0 withPlusSign) in cases where a small number is rounded to zero. */
+    strictZeroUnsigned?: boolean;
+
```

Comment:
how about something like `strictZero`?  Captures the fact that this relates to interpretation of numbers rounding to zero generally, both for the purposes of zeroDisplay and the signing.   But I think the name should imply the reverse sense.  That is 'strict' means that only true zero gets treated like zero

Reply:
other than that, implementation looks good!